Makefile

```diff
@@ -34,7 +37,7 @@ ifeq ($(shell git tag),)
 VERSION := $(shell echo "v0.0.0-$$(git rev-list HEAD --count)-g$$(git describe --dirty --always)" | sed 's/-/./2' | sed 's/-/./2')
 else
 # use tags
-VERSION := $(shell git describe --dirty --always --tags | sed 's/-/./2' | sed 's/-/./2')
+VERSION := $(shell git describe --dirty --always --tags --exclude 'helm*' | sed 's/-/./2' | sed 's/-/./2')
```

Comment:
When we create a helm release a tag `helm-chart-x.y.z` gets created which will be used as container image tag. We shouldn't use these tags to version container images. See image listing here: https://github.com/external-secrets/external-secrets/pkgs/container/external-secrets/versions